Lake/CLI/Main.lean

```diff
@@ -202,7 +206,10 @@ The `print-paths` command is used internally by Lean 4 server.
 def printPaths (imports : List String := []) : CliStateM PUnit := do
   let (lean, lake) ← getInstall
   let configFile := (← getRootDir) / (← getConfigFile)
-  if (← configFile.pathExists) then
+  if (← IO.getEnv cannotParseLakefileEnv) matches some .. then
+    IO.eprintln "Error parsing 'lakefile.lean'.  Please restart the lean server after fixing the lakefile."
```

Comment:
```suggestion
    IO.eprintln s!"Error parsing '{configFile}'.  Please restart the lean server after fixing the Lake configuration file."
```

Note that `configFile` can be something other than `lakefile.lean` (it can be  user specified).

Reply:
Ah I see, didn't realize this was a thing.  This is set by a command-line flag.